dbmate/db/migrations/20200419091342_device_telemetry.sql

```diff
@@ -6,7 +6,7 @@ create table device_telemetry (
     device_id varchar(255) NOT NULL,
     creation_timestamp timestamp NOT NULL DEFAULT CURRENT_TIMESTAMP,
     FOREIGN KEY(username) REFERENCES users(username) ON DELETE CASCADE,
-    FOREIGN KEY(recipient_username) REFERENCES users(username) ON DELETE CASCADE
+    CONSTRAINT device_recipient_fk FOREIGN KEY(recipient_username) REFERENCES users(username) ON DELETE CASCADE
```

Comment:
HI @JasterV we can not modify existing files, instead, we need to do the migration in a different file.

Reply:
I'm not modifying the structure of the table/migration there, just giving a name to the fk constraint so I can remove that constraint on the new migrationgrin

I could not find a way to remove an unnamed constraint without removing the column:/ (which could be another option)

Reply:
@JasterV I think that we will have to find some workaround, maybe it is possible to build a sub query to find the name that Postgres generates?

Reply:
Actually it seems to be consistently named `device_telemetry_recipient_username_fkey` so I think that this change could be removed. Something like this
```
ALTER TABLE device_telemetry DROP CONSTRAINT device_telemetry_recipient_username_fkey;
```

Reply:
Perfect! Let's do that then ^^